engine/Render/Renderer/Camera.h

```diff
@@ -0,0 +1,67 @@
+#ifndef RENDER_RENDERER_CAMERA_INCLUDED
+#define RENDER_RENDERER_CAMERA_INCLUDED
+
+#include <glm.hpp>
+#include <Reflection/macros.h>
+
+namespace Engine
+{
+    class Transform;
+
+    class REFL_SER_CLASS(REFL_BLACKLIST) Camera
+    {
+        REFL_SER_BODY(Camera)
+    public:
+        Camera();
+        virtual ~Camera() = default;
+
+        /// @brief Acquire view matrix that transforms world coordinate to eye coordinate
+        /// @return view matrix
+        glm::mat4 GetViewMatrix();
+
+        /// @brief Acquire projection matrix that transforms eye coordinate to clip space coordinate.
+        /// Handness is not flipped by projection matrix.
+        /// It should be flipped in vertex shader by postmultipling vec4(1.0, 1.0, -1.0, 1.0).
```

Comment:
This docstring seems to be a leftover from ancient times :(
```suggestion
        /// OpenGL/Vulkan handness is ensured by the projection matrix,
        /// and should not be flipped again in vertex shaders.
```